plugins/github-dev/skills/cr-fix/scripts/cr-cli-spawn.sh

```
@@ -0,0 +1,24 @@
+# Prints terminal JSON marker on stdout: {"jsonl":"<path>","exit":N,"emitted_complete":bool}
+set -euo pipefail
+
+: "${BASE:?BASE base-branch required}"; : "${PR_NUM:?}"; : "${ITER:?}"
+: "${CONFIG_FILES:=CLAUDE.md AGENTS.md}"
+
+OUT="/tmp/cr-cli-review-${PR_NUM}-iter${ITER}.jsonl"
+: > "$OUT"
+
+# shellcheck disable=SC2086  # CONFIG_FILES is intentionally word-split
+set +e
+coderabbit review --agent --type committed --base "$BASE" --config $CONFIG_FILES > "$OUT" 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)